src/FondOfSpryker/Zed/ThirtyFiveUp/Persistence/Propel/Schema/thirty_five_up.schema.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0"?>
+<database xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" name="zed"
+          xsi:noNamespaceSchemaLocation="http://static.spryker.com/schema-01.xsd"
+          namespace="Orm\Zed\ThirtyFiveUp\Persistence" package="src.Orm.Zed.ThirtyFiveUp.Persistence">
+
+    <table name="fos_thirty_five_up_order" phpName="ThirtyFiveUpOrder" identifierQuoting="true">
+        <column name="id_thirty_five_up_order" required="true" type="INTEGER" autoIncrement="true" primaryKey="true"/>
+
+        <column name="fk_order_id" required="false" type="INTEGER"/>
```

Comment:
Does it make sense to use the name `fk_sales_order` for consistency or what is the table name releated to this FK?

Reply:
ur right